### PR TITLE
fix!: 💥Parameter changes of the `Showcase` and `ShowCaseWidget` class

### PR DESCRIPTION
**Pull request by HappyMakadiyaS**
# Description

This PR consist changes of parameter names in Showcase and ShowcaseWidget class.
Showcase class parameter:
- shapeBorder -> targetShapeBorder
- radius -> targetBorderRadius
- contentPadding ->tooltipPadding
- showcaseBackgroundColor ->tooltipBackgroundColor
- overlayPadding -> targetPadding
- tipBorderRadius -> tooltipBorderRadius

ShowCaseWidget class parameter:
- autoPlayLockEnable -> enableAutoPlayLock

Some of the parameter of Showcase.withWidget() has been moved from parameter list to initializer list which are 
title, description, titleAlignment, descriptionAlignment, titleTextStyle, descTextStyle, tooltipBackgroudColor, textColor, tooltipBorderRadius, tooltipPadding

## Checklist

- [x] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `docs:` etc).
- [x] I have followed the [Contributor Guide] when preparing my PR.
- [-] I have updated/added tests for ALL new/updated/fixed functionality.
- [-] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [x] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?

- [x] Yes, this PR is a breaking change.
- [-] No, this PR is not a breaking change.

[Contributor Guide]: https://github.com/SimformSolutionsPvtLtd/flutter_showcaseview/blob/master/CONTRIBUTING.md
[Conventional Commit]: https://conventionalcommits.org

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,13 +67,13 @@ Showcase(
   titleAlignment: TextAlign.start,
```
Comment (vatsaltanna) [inline README.md:67]: add seprate section of breaking changes

**Comment by ShwetaChauhan18** [comment]
Currently, PR title validation workflow not merged so it will not check title but from my side I am suggesting this.
PR title should be: 

fix: 💥Parameter changes of the `Showcase` and `ShowCaseWidget` class

**Comment by ShwetaChauhan18** [comment]
If you are mentioning the class name anywhere, use `CLASSNAME` as this symbol.

File: README.md [hunk 2]
```diff
@@ -11,7 +11,35 @@ A Flutter package allows you to Showcase/Highlight your widgets step by step.
...
+<tr><td>autoPlayLockEnable</td><td>enableAutoPlayLock</td></tr>
+<tr><td>shapeBorder</td><td>targetShapeBorder</td></tr>
+<tr><td>showcaseBackgroundColor</td><td>tooltipBackgroundColor</td></tr>
+<tr><td>contentPadding</td><td>tooltipPadding</td></tr>
+<tr><td>overlayPadding</td><td>targetPadding</td></tr>
+<tr><td>radius</td><td>targetBorderRadius</td></tr>
+<tr><td>tipBorderRadius</td><td>tooltipBorderRadius</td></tr>
+</table>
+
+Unused parameters of `Showcase.withWidget()` which are not impacting view of showcase widget that has been removed from parameter list of the named constructor `Showcase.withWidget()`.
```
Comment (vatsaltanna) [inline README.md:28]: Unused parameters of `Showcase.withWidget()` which do not impact the UI of showcase have been removed from the parameter list of the named constructor `Showcase.withWidget()`.

**Review by vatsaltanna — APPROVED** [review]

**Pull request merged**